ginga/rv/plugins/Catalogs.py

```diff
@@ -67,13 +71,15 @@ def __init__(self, fv, fitsimage):
         canvas = self.dc.DrawingCanvas()
         canvas.enable_draw(True)
         canvas.set_drawtype(self.drawtype, color='cyan', linestyle='dash')
-        canvas.set_callback('cursor-down', self.btndown)
-        canvas.set_callback('cursor-up', self.btnup)
         canvas.set_callback('draw-event', self.draw_cb)
+        canvas.add_draw_mode('select', down=self.btndown,
+                             move=self.btndown, up=self.btnup)
```

Comment:
Actually, I spoke too soon. I cannot find any actual implementation in other `rv` plugins that use this "pick" callback. Maybe it is best I leave that to you *after* this PR is merged. I'll remove `btndown` as you suggested.